automation/project/Dependencies.scala

```diff
@@ -57,7 +57,7 @@ object Dependencies {
     "org.scalatest"       %%  "scalatest"     % "3.2.2"   % Test,
     "org.seleniumhq.selenium" % "selenium-java" % "3.8.1" % Test,
     "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
-    "org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-04a7a76b"
+    "org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-dbe3ca9c"
```

Comment:
rawls-model update needed for change to include landingZoneId in the `AzureManagedAppCoordinates`.